### ISSUE TITLE
Distro packages

### ISSUE THREAD

**Issue by dhewg** [dhewg/nine#9]
We cannot easily provide binaries for end users (see #4), but using Gallium Nine should still be as easy as possible.
Get this project packaged for various distros, so users can just use their well-known package manager.

Setting up a `WINE` prefix could be unified and improved across distros. Currently there's the very basic `nine-install.sh`, which is unlikely to satisfy the requirements for everyone. Feedback is very much welcome!

**Comment by dhewg** [comment]
Arch: https://aur.archlinux.org/packages/gallium-nine-git/

**Comment by dhewg** [comment]
Gentoo: @chewi seems to work on it

**Comment by dhewg** [comment]
Debian/Ubuntu: @aviau might be interested? (DXVK maintainer, which is very similar)

**Comment by aviau** [comment]
I'd be more than willing to sponsor an upload to Debian. As for maintaining it myself, I would also be interested but I don't have time right now. Please include me in all future Ubuntu/Debian discussions however.

Also, I wonder if the community needs to settle on a way to install prefix "addons".

Could there be a standard directory for them?

Could they be installed locally via a winetricks command?

This might be related (my dxvk install script for Debian users):
 - https://sources.debian.org/src/dxvk/latest/debian/dxvk-setup/dxvk-setup/

As discussed recently with someone else, my script currently:
 - Does not support custom wine versions
 - Does complains when using wine-stable, but it shouldn't.

Just my 2 cents.

**Comment by chewi** [comment]
@aviau, you may want to see [how I handled it in Gentoo](https://gitweb.gentoo.org/repo/gentoo.git/tree/app-emulation/gallium-nine-standalone?id=98b911b67554ddbecaa92b471a6cbdf43716766a), given that we allow multiple Wine implementations and versions to be installed simultaneously. Rather than use `nine-install.sh` or similar, I installed the dll and exe to a global location and changed the exe to always look for the dll in that location.

**Comment by dhewg** [comment]
Debian package request: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=919863

**Comment by montvid** [comment]
Would be nice to have a debian/ubuntu build! I really like gallium nine! Do people still bugfix it? Is a feature complete dx9 implementation?

**Comment by montvid** [comment]
Okay, I can't compile this because ubuntu 18.04 does not have libd3dadapter9-mesa-dev only Cosmic 18.10 has it. What to do?

**Comment by dhewg** [comment]
@montvid yes, gallium nine is still maintained. For bionic, you could use [padoka ppa](https://launchpad.net/~paulo-miguel-dias/+archive/ubuntu/pkppa/), but for whatever reason `libd3dadapter9` seem to be part of `libgl1-mesa-dri` and `mesa-common-dev` there.

**Comment by dhewg** [comment]
@jcstryker, @chewi: we moved to a new home, and I just released v0.2. When you update your packages, please use the new home: https://github.com/iXit/wine-nine-standalone/releases/tag/v0.2

**Comment by dhewg** [comment]
I'll close this since we moved
